### PR TITLE
Modify task card to have different colour if task is completed

### PR DESCRIPTION
**Pull request by turtle96**
sample:
image

@CS2103AUG2016-W15-C3/testing 
@CS2103AUG2016-W15-C3/code-quality 
@CS2103AUG2016-W15-C3/documentation

### CODE REVIEW

**Comment by turtle96** [comment]
part of #54

**Comment by turtle96** [comment]
@CS2103AUG2016-W15-C3/testing
@CS2103AUG2016-W15-C3/code-quality
@CS2103AUG2016-W15-C3/documentation

File: src/main/java/seedu/taskell/ui/CalendarView.java [hunk 1]
```diff
@@ -46,6 +44,8 @@ private void setAgendaProperties() {
         agenda.setStyle("-fx-font-size: 12pt");
     }
     
+    /** gets current filtered task list from model and updates agenda's data
```
Comment (marcusngwenjian) [inline src/main/java/seedu/taskell/ui/CalendarView.java:47]: @turtle96  Comment not consistent. Should start with captial G

**Comment by marcusngwenjian** [comment]
Documentation, Testing, Code Qualiy reviewed.

**Pull request merged**